lib/Runtime/ByteCode/ByteCodeWriter.cpp

```diff
@@ -1525,6 +1525,7 @@ namespace Js
 #endif
         case OpCode::StObjSlot:
         case OpCode::StObjSlotChkUndecl:
+        case OpCode::StPropIdArrFromVar:
```

Comment:
Is this left over from the previous slot implementation before switching to PropertyIdArray, or do you still need it? Same question for the next one below.

Reply:
yup, leftover

---
In reply to: [204247582](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r204247582) [](ancestors = 204247582)